ibis/backends/base/sqlglot/compiler.py

```diff
@@ -868,14 +868,14 @@ def visit_JoinLink(self, op, *, how, table, predicates):
             "anti": "left",
             "cross": None,
             "outer": "full",
-            "asof": "left",
+            "asof": "asof",
             "any_left": "left",
             "any_inner": None,
         }
         kinds = {
             "any_left": "any",
             "any_inner": "any",
-            "asof": "asof",
+            "asof": "left",
```

Comment:
This swap seems incorrect. Was it necessary for one of the backends?

Reply:
We need `ASOF LEFT JOIN` and prior to the change it was generating `LEFT ASOF JOIN` which is not accepted by duckdb.